contents/uix/design-systems.md

```diff
@@ -0,0 +1,108 @@
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Design Systems
+
+**Author(s): [Tan Heng Yeow](https://github.com/tanhengyeow)**
+
+## What is a Design System
+
+Design Systems has been a trending topic for the past 2 years. The following definition best encapsulates the essence of a Design System.
+> A Design System is the Single Source of Truth which groups all the elements that will allow the teams to design, realize and develop a product. <sub>--[Blog post from UX Collective](https://uxdesign.cc/everything-you-need-to-know-about-design-systems-54b109851969)</sub>
+
+The typical structure of a Design System looks like this:
+![Typical structure of a Design System](https://cdn-images-1.medium.com/max/2600/1*wSHUJh069L618oXqVZYDtA.png)
+
+Below is a brief explanation of the parts that make up a Design System. The explanation includes reference to a living example, [Polaris](https://polaris.shopify.com/), Shopify's Design System.
+- `Rules`: Rules include abstract elements such as brand values, mindset and shared beliefs. You can refer to Polaris' [product experience principles](https://polaris.shopify.com/patterns-and-guides/product-experience-principles#navigation).
+- `Pattern Library`: A Pattern Library integrates functional components and their usage. You can refer to Shopify's [product component library](https://github.com/Shopify/polaris-react).
+- `Building Blocks`: Building Blocks refer to documentation consisting of the collection of interface elements. You can refer to Polaris' documentation which describes their [components](https://polaris.shopify.com/components/get-started).
+- `Style Guide`: A Style Guide focuses on graphic styles such as colours, fonts, illustrations etc and their usage. You can refer to Polaris' [style guide](https://polaris.shopify.com/design/colors).
+
+## Why should we use a Design System?
+
+![Typical Design and Development process](https://cdn-images-1.medium.com/max/2600/1*wFOwLNi-xGBfyGnssynQaQ.png)
+
+Consider the typical design and development process. Designers come up with sketches, wireframes and hand them over to the developers to implement them through code. Here's the problem.
+
+Designers are looking at multiple directions and solutions, constantly comparing and tweaking their design. On the other hand, developers are ultimately accountable for shipping code. They focus on making sure that the correct architecture is in place, thinking about security and the performance of the product, hoping that the designers already have all the design details thought out.
+
+Over time, it is likely that the many teams working on different parts of the product will create inconsistencies in the product. For example, HubSpot [explained how they found inconsistencies](https://product.hubspot.com/blog/how-building-a-design-system-empowers-your-team-to-focus-on-people-not-pixels) in their interface after auditing their User Interface (UI) components.
```

Comment:
a little verbose, the explaination of why they shuld be used.

Reply:
Thanks for the review 😄 

I'm not too sure which part you were referring to. Are you saying the whole paragraph is too verbose or this part `[explained how they found inconsistencies]` that is lengthy?

Reply:
The whole part about different people churning out different products